cli/errors.rs

```diff
@@ -41,3 +61,14 @@ pub(crate) fn get_error_class_name(e: &AnyError) -> &'static str {
       );
     })
 }
+
+pub(crate) fn clone_any_error(error: &AnyError) -> AnyError {
```

Comment:
I have some reservations about this function.

Reply:
Yeah... I first had a function in `core/error.rs` that was `maybe_clone_custom_error(error: &AnyError) -> Option<AnyError>` and it grew out of hand to meet needs of call sites in `cli/module_graph.rs`. I'll move it there with a new name.